sql/core/src/main/scala/org/apache/spark/sql/DataStreamWriter.scala

```diff
@@ -84,6 +84,17 @@ final class DataStreamWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Specifies a name to the [[ContinuousQuery]] to be started. This name must be unique among
```

Comment:
"Specifies a name for this [[ContinuousQuery]]. The name must be unique for any active query, and will be auto assigned if unspecified."